metabase/metabase
# Scope HTML styles and CSS custom properties

Epic https://github.com/metabase/metabase/issues/38811
Sub-task https://github.com/metabase/metabase/issues/41256

### Description

Scope HTML styles and CSS custom properties in `base.module.css`

### How to verify
1. Test that the app style doesn't look weird. The `body` node should contain the same style plus CSS custom properties from `:root`
    ![image](https://github.com/metabase/metabase/assets/1937582/ab8667a2-dc3b-4727-ac44-89be60bdbc17)
2. Test that the [example app](https://github.com/metabase/embedding-sdk-customer-zero/tree/pre-alpha) inherit all the styles from `base.styled.ts` and is scoped under the root SDK div.
    ![Screenshot 2024-04-17 at 10 15 30 PM](https://github.com/metabase/metabase/assets/1937582/7493031b-430c-4d32-8ad5-49bfcbd16224)

    To setup the example app:
    1. I set the dependencies in the example app to the locally built SDK path `"@metabase/embedding-sdk-react": "file:../metabase/resources/embedding-sdk"` 👈 change the path to whatever the directory structure on your computer is. In my case, both the example app and metabase repos are on the same level.
    2. Remove `.npmrc`
    4. Run `yarn install-deps && yarn start`

### Checklist

- [ ] ~Tests have been added/updated to cover changes in this PR~ Only change CSS
